class/defaults.yml

```diff
@@ -1,4 +1,41 @@
 parameters:
   emergency_credentials_controller:
     =_metadata: {}
-    namespace: syn-emergency-credentials-controller
+    namespace: appuio-emergency-credentials-controller
+    namespaceMetadata:
+      annotations:
+        openshift.io/node-selector: "node-role.kubernetes.io/infra="
+      labels:
+        openshift.io/cluster-monitoring: "true"
+
+    images:
+      emergency_credentials_controller:
+        registry: ghcr.io
+        image: appuio/emergency-credentials-controller
+        tag: master
+      kube_rbac_proxy:
+        registry: gcr.io
+        image: kubebuilder/kube-rbac-proxy
+        tag: v0.14.1
+
+    manifests_version: ${emergency_credentials_controller:images:emergency_credentials_controller:tag}
+
+    kustomize_input:
+      namespace: ${emergency_credentials_controller:namespace}
+
+    cluster_admin_role: cluster-admin
+
+    emergency_accounts: {}
+
+    alerts:
+      EmergencyAccountTokenExiring:
```

Comment:
Currently duplicated from upstream. Did not figure out a quick way to patch the Kustomize output. Should not change that often anyways.